### DISCUSSION TITLE
How can I add Bluesky-powered comments to my website?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#1003] [Ask a question] [answered]
Today I read this article:

[https://www.coryzue.com/writing/bluesky-comments/
](https://www.coryzue.com/writing/bluesky-comments/)

It teaches us how to use Bluesky as a comment system for our blogs. Then, I made a test with Yellow, adding the CSS file inside the tag` <head>.` (/system/layouts/header.html)
Then I went to the "content" directory", then "3-blog", and edited the following file: 2020-12-06-made-for-people.md

My header now looks like this:

```
---
Title: Made for people
Published: 2020-12-06
Author: Datenstrom
Layout: blog
Tag: Example
bluesky_post_uri: "https://bsky.app/profile/my-name.bsky.social/post/numbersandlettersexample"
---
```

I finished it adding the codes in the steps 3 (Add source maps for React) and 4 (Import and use the library and any other functions you need in an ES module script) at the footer in this same same post, ending with the 

`<div id="bluesky-comments">Comments</div> .`

Unfortunately, the comments are not loading:

[http://www.blogri.link/blog/made-for-people](http://www.blogri.link/blog/made-for-people)

Well, if someone know how I can fix this, it would be great.

**Comment by schulle4u** [comment, marked as answer]
First of all, could you please edit your discussion and write the title as a question so that it meets our community guidelines? 

It's hard to solve the problem without the Yello-specific code. The linked example probably won't work without modifications. You may have specified `bluesky_post_uri` in your page settings, but maybe you haven't processed this setting somewhere in the initialization code. So the script does not know for what page to display the comments. However I don't know enough about Bluesky to answer this correctly. Also, there may be a problem with inline javascript if it is filtered out by Yello.

**Reply by ghost** [reply]
Hi schulle4u!

Thank you for your answer.
No problem, I had no idea that it would be a hard task ...

A great week for you!

**Comment by annaesvensson** [comment]
Usually you need to include some CSS/Javascript files in the header of your website. 

If you like to do this manually in Datenstrom Yellow then edit the file `system/layouts/header.html`. Check in your web browser that the CSS/Javascript files are successfully loaded. Some libraries will look for a specific HTML element in the web page, for example to know where to show comments, for the mentioned library you will also have to add the element `<div id="bluesky-comments"></div>` to your blog layout file. For more details see https://github.com/czue/bluesky-comments.

If you prefer a simple solution then use an extension. All of the above can be automated with an extension for Datenstrom Yellow. Here are some existing commenting systems for Datenstrom Yellow: [yellow-comment](https://github.com/GiovanniSalmeri/yellow-comment), [yellow-disqus](https://github.com/GiovanniSalmeri/yellow-disqus), [yellow-cusdis](https://github.com/schulle4u/yellow-cusdis) and if you want to make a new one for Bluesky [there's also an experimental extension for developers](https://github.com/schulle4u/yellow-helloworld). 

It's super easy to add a commenting system. 😁

**Reply by ghost** [reply]
Hi Annaesvensson!

Thank you very much for your answer.

A great week for you!
